src/main/lrsql/spec/reaction.clj

```diff
@@ -232,3 +232,19 @@
 
 (def error-reaction-ret-spec
   (s/keys :req-un [:lrsql.spec.reaction.error-reaction/result]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Params
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(def create-reaction-params-spec
+  (s/keys :req-un [::ruleset
+                   ::active]))
+
+(def update-reaction-params-spec
+  (s/keys :req-un [::reaction-id
+                   (or ::ruleset
```

Comment:
Why is this an or? You can't update both?

Reply:
I believe this just requires that _one_ of them be present, though it will also let them both be.